lens_editor/app.py

```diff
@@ -29,119 +30,119 @@
 
 
 class MainWindow(QMainWindow):
-    def __init__(self, initial_path=""):
+    def __init__(self, initial_path="") -> None:
```

Comment:
python3.11中增加了Self类, 在python3.11中运行此项目进行测试
```suggestion
    def __init__(self, initial_path="") -> Self:
```